src/functions/uploadToRSIS/application/__mocks__/result-uploads.mock.ts

```diff
@@ -0,0 +1,1093 @@
+
+import { ResultUpload, InterfaceType } from './result-client.mock';
+
+const staffNumber = '000002';
+const tcId = 3024;
+
+const generateRandomText = (length: number): string => {
```

Comment:
Please add comment: This is used to test text being written to Oracle CLOB columns.

Reply:
Added the comment.